modules/publication/ajax/FileDelete.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Publication file deletion script
+ *
+ * Deletes a specified file by upload ID (PublicationUploadID)
+ *
+ * PHP Version 7
+ *
+ * @category Loris
+ * @package  Publication
+ * @author   David <dblader.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
```

Comment:
This script is still missing a permission check and doing a hard delete with no way to recover files that are accidentally deleted.